python/tvm/relax/vm.py

```diff
@@ -585,6 +585,9 @@ def foo(x: Tensor((3, 4), "float32"), y: Tensor((3, 4), "float32")):
     passes.append(relax.transform.RewriteDataflowReshape())
     passes.append(relax.transform.ToNonDataflow())
     passes.append(relax.transform.CallTIRRewrite())
+    if target.kind.name == "cuda":
+        passes.append(relax.transform.CanonicalizeBindings())
+        passes.append(relax.transform.RewriteCUDAGraph())
```

Comment:
One quick question about the pass ordering, considering the next rebase of mlc-ai/relax. In tlc-pack the memory planning pass was applied immediately after CallTIRRewrite. https://github.com/tlc-pack/relax/blob/relax/python/tvm/relax/vm.py#L613

I feel like the CUDA graph rewrite should be applied prior to the memory planning. Could you help take a look and confirm this?